Makefile

```diff
@@ -197,7 +197,10 @@ run-tests-within-container:
 
 ifeq ($(COVERAGE),1)
 COVERDB_SUFFIX ?=
-COVEROPT ?= -MDevel::Cover=-select_re,'^/lib',+ignore_re,lib/perlcritic/Perl/Critic/Policy,-coverage,statement,-db,cover_db$(COVERDB_SUFFIX),
+# We use JSON::PP because there is a bug producing a (harmless) 'redefined'
+# warning when using Devel::Cover and Cpanel::JSON::XS
+# https://progress.opensuse.org/issues/90371
+COVEROPT ?= -mJSON::PP -MDevel::Cover=-select_re,'^/lib',+ignore_re,lib/perlcritic/Perl/Critic/Policy,-coverage,statement,-db,cover_db$(COVERDB_SUFFIX),
```

Comment:
No comma there? 🤔

Reply:
not sure where you mean

Reply:
The string was using `,` to separate arguments and you used ` `. No idea if it makes a different, just stood out to me that you didn't for example change everything to use spaces.